assets/src/edit-story/elements/video/frame.js

```diff
@@ -14,14 +14,68 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import { useState } from 'react';
+import styled from 'styled-components';
+
 /**
  * Internal dependencies
  */
 import StoryPropTypes from '../../types';
 import MediaFrame from '../media/frame';
+import { elementFillContent } from '../shared';
+import { useStory } from '../../app/story';
+import { ReactComponent as PlayIcon } from '../../icons/play.svg';
+import { ReactComponent as PauseIcon } from '../../icons/pause.svg';
+
+const Play = styled(PlayIcon)`
+  width: 48px;
+  height: 48px;
+`;
+const Pause = styled(PauseIcon)`
+  width: 48px;
+  height: 48px;
+`;
+
+const Wrapper = styled.div`
+  ${elementFillContent}
+`;
+
+const ButtonWrapper = styled.div`
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  cursor: pointer;
+`;
 
 function VideoFrame({ element }) {
-  return <MediaFrame element={element} />;
+  const { id, isPlaying } = element;
+  const [hovered, setHovered] = useState(false);
+
+  const {
+    actions: { updateElementById },
+  } = useStory();
+
+  const handlePlayPause = () => {
+    updateElementById({ elementId: id, properties: { isPlaying: !isPlaying } });
+  };
+
+  return (
+    <Wrapper
+      onPointerEnter={() => setHovered(true)}
+      onPointerLeave={() => setHovered(false)}
+    >
+      <MediaFrame element={element} />
+      {hovered && (
+        <ButtonWrapper onClick={handlePlayPause}>
+          {isPlaying ? <Pause /> : <Play />}
+        </ButtonWrapper>
+      )}
```

Comment:
Could this use `CSSTransitionGroup` and fade the button in/out over .2s with standard easing? It's a bit abrupt as it is now.

Reply:
Done in https://github.com/google/web-stories-wp/pull/720/commits/2a95296e1996717467001e7f07fed839f856b00c